### ISSUE TITLE
Image caching problem

### ISSUE THREAD

**Issue by Terxeno** [desertblade/PiPup#2]
If you send an image with the same name, the same image is displayed on TV until you change the file name.

**Comment by SankeerthBoddu** [comment]
Yes, i have the same problem.... did you figure out ? I used a delay of 3 secs.. still same.. 
In the browser, i do however see the image with newer timestamp

**Comment by 8666** [comment]
maybe add random value at the end of request? ..............&random=43523534 (timestamp)

**Comment by Terxeno** [comment]
> maybe add random value at the end of request? ..............&random=43523534 (timestamp)

Yes you are right! Not ideal but working.

**Comment by SankeerthBoddu** [comment]
cool, will try with random.

also - was reading about this caching - _it’s not a caching issue. If you want the image to update and you’re using the same image name, you need to display the image as a camera._
https://community.home-assistant.io/t/caching-issue-with-camera-snapshot-notification/307822/8?u=sankeerth
